terraform/modules/prometheus/input.tf

```diff
@@ -12,20 +14,21 @@ variable disk_quota { default = 1024 }
 
 variable influxdb_service_instance_id {}
 
-variable alert_rules {
-  default = ""
-}
+variable alert_rules { default = "" }
 
-variable config_file {
-  default = ""
-}
+variable extra_scrape_config { default = "" }
 
 locals {
   template_variable_map = {
     paas_prometheus_exporter_endpoint = var.paas_prometheus_exporter_endpoint
+    redis_prometheus_exporter_endpoint= var.redis_prometheus_exporter_endpoint
     alertmanager_endpoint             = var.alertmanager_endpoint
     paas_prometheus_exporter_name     = "paas-prometheus-exporter-${var.monitoring_instance_name}"
     include_alerting                  = var.alert_rules == "" ? false : true
+    include_redis_exporter            = var.redis_prometheus_exporter_endpoint == "" ? false : true
+    include_scrapes                   = var.extra_scrape_config == "" ? false : true
+    scrapes                           = var.extra_scrape_config
```

Comment:
This doesn't do anything. Can we remove it and call `var.extra_scrape_config` directly?